modules/govuk/manifests/apps/ckan/cronjobs.pp

```diff
@@ -30,8 +31,15 @@
     ensure         => $ensure_solr_reindex,
     paster_command => 'search-index rebuild -o',
     plugin         => 'ckan',
-    hour           => '7',
+    hour           => '5',
```

Comment:
How come the time is changing for this? With my limited understanding of CKAN, I don't know how this relates to adding the csv endpoint.

Reply:
Changed it because `csw` needs solr reindexed for it to work, if we left it at 7 there is a greater chance that the sync between `csw` and `ckan` would happen later after the working day starts and it would be better if the data is synced on the day otherwise its possible that changes will happen a day late. 
I've discussed with Tom and the original 7 am was an arbitrary time so was ok to change.

Reply:
I can see that this change is now being made in a separate commit, with this reason being given.